pkgs/development/python-modules/triton/default.nix

```diff
@@ -241,25 +241,31 @@ buildPythonPackage {
             assert output_torch.sub(output_triton).abs().max().item() < 1e-6
             print("Triton axpy: OK")
         '';
+  };
 
-    gpuCheck = stdenv.mkDerivation {
-      pname = "triton-pytest";
-      inherit (triton) version src;
-
-      nativeBuildInputs = [
-        (python.withPackages (ps: [
-          ps.scipy
-          ps.torchWithCuda
-          ps.triton-cuda
-        ]))
-      ];
-
-      nativeCheckInputs = [ pytestCheckHook ];
-      dontBuild = true;
-      doCheck = true;
-      requiredSystemFeatures = [ "cuda" ];
-      installPhase = "touch $out";
-    };
+  passthru.gpuCheck = stdenv.mkDerivation {
```

Comment:
Note: this is left outside `passthru.tests` on purpose, because of https://github.com/NixOS/ofborg/issues/678